kube/kd/deployment.yaml

```diff
@@ -10,18 +10,18 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: hocs-toolbox
+      name: hocs-migration-toolbox
   template:
     metadata:
       labels:
-        name: hocs-toolbox
+        name: hocs-migration-toolbox
         version: {{.VERSION}}
-        role: hocs-backend
+        role: hocs-cms-migration
```

Comment:
Changing this role will mean the container will no longer match selectors within the network policies. That's probably what you want, but bear that in mind.